docs/libcudacxx/standard_api/utility_library.rst

```diff
@@ -26,6 +27,9 @@ the information about the individual features for details.
    * - Header
      - Content
      - Availability
+   * - :ref:`libcudacxx-standard-api-utility-bitset`
+     - Fixed-size sequence of bits
+     - CCCL to be released / CUDA to be released
```

Comment:
I believe we can confidently say this will be in CCCL 2.7 and drop the reference to the CTK until and if a new CTK comes out
```suggestion
     - CCCL 2.7.0
```